userspace/sysdig/sysdig.cpp

```diff
@@ -185,15 +185,16 @@ static void usage()
 "                    If no data format is specified, this can be used with -W flag to\n"
 "                    create a ring buffer of events.\n"
 " -h, --help         Print this page\n"
-" -I <inputname>[:<inputargs>], --input <inputname>[:<inputargs>]\n"
+" -I <inputname>[:<inputargs>], --input <inputname>[:I<initconf>][:O<openparams>]\n"
 "                    (PREVIEW feature, subject to change)\n"
 "                    capture events using the plugin with name inputname, passing to the \n"
-"                    plugin the inputargs string as parameters.\n"
-"                    The format of inputargs is controller by the plugin, refer to each\n"
+"                    plugin the initconf string as init parameters, and openparams as open parameters.\n"
```

Comment:
It's not too clear to me what an init parameter is and what an open parameter is and which is the difference. Perhaps we could link to the documentation page that explains it?

Reply:
Good idea!

Reply:
Does this look better?
> -I <inputname>[:<inputargs>], --input <inputname>[:I<initconf>][:O<openparams>]
(PREVIEW feature, subject to change)
capture events using the plugin with name inputname, passing to the 
plugin the initconf string as init parameters, and openparams as open parameters.
The format of initconf and openparams is controller by the plugin, refer to each
plugin's documentation to learn about it.
See https://falco.org/docs/plugins/plugin-api-reference/#ss-plugin-t-plugin-init-const-char-config-int32-t-rc-required-yes
and https://falco.org/docs/plugins/plugin-api-reference/#ss-instance-t-plugin-open-ss-plugin-t-s-const-char-params-int32-t-rc-required-yes for more infos.
The event sources available for capture vary depending on which 
plugins have been installed. You can list the plugins that have been 
loaded by using the -Il flag.
Example for dummy plugin: sysdig -I dummy:"O{\"start\":1,\"maxEvents\":10}:I{\"jitter\":50}"